app/api.py

```diff
@@ -791,3 +798,21 @@ def get_element_text(self, doc, tag_name):
             return text.encode('utf-8')
         else:
             return None
+
+    #Send hit to GA
+    def gamp(e,ec):
+        ga_cid = uuid.uuid4()
+        params = urllib.urlencode({
+            'v': 1,
+            'tid': 'UA-17770602-2',
+            'cid': ga_cid,
+            't': 'event',
+            'ec': ec,
+            'ea': 'SMS',
+            'dp': '/sms_action'
+        })
+        url = 'http://www.google-analytics.com/collect'
+        try:
+            f = urllib2.urlopen(url, params)
+        except urllib2.URLError:
+            logging.exception('Caught exception fetching url')
```

Comment:
Can you make the message clearer to say that it failed to send Measurement Protocol?